Osiris/Hacks/Visuals.cpp

```diff
@@ -129,6 +129,12 @@ void Visuals::playerModel(FrameStage stage) noexcept
     }
 }
 
+void Visuals::rdrawothermodels() noexcept
+{
+    static auto rdrawothermodels = interfaces->cvar->findVar("r_drawothermodels");
+    rdrawothermodels->setValue(config->visuals.rdrawothermodels ? 100 : 0);
```

Comment:
why make a function for this, can't you do this directly in hooks?

Reply:
zzzz why dont you do it yourself then, stop complaining on others work

Reply:
@mkot2 bro what's the point its working lol

Reply:
> 
> 
> zzzz why dont you do it yourself then, stop complaining on others work

how am i complaining bro i'm just sayin